core/runtime/ops_rust_to_v8.rs

```diff
@@ -278,7 +278,13 @@ to_v8_retval!((f32, f64): |value, rv| rv.set_double(value as _));
 // Heavier primitives with no retval shortcuts
 //
 
-to_v8!((*const c_void, *mut c_void): |value, scope| v8::External::new(scope, value as _));
+to_v8!((*const c_void, *mut c_void): |value, scope| {
```

Comment:
Perfect.. let's implement `to_v8_retval` as well since we have the extra-fast path of null.

Reply:
I'm not sure how to do this since there is no `ReturnValue.set_external` and `to_v8_retval` does not pass scope to its callback, which is necessary for constructing an `External`.

https://v8.github.io/api/head/classv8_1_1ReturnValue.html

Reply:
My bad, we're good with this!